### PR TITLE
Docs: Clarinet remote content docs

### PR DESCRIPTION
**Pull request by LakshmiLavanyaKasturi**
- Added documentation files to the `docs` folder.  
- Added new images to the docs/images folder.
- Added the following structure into the docs folder. 
image

**Motivation**:
Remove duplication of content in the docs repo.

- Implemented Approach -4 from [this](https://www.notion.so/hiropbc/Docusaurus-plugins-to-get-remote-content-to-docs-repo-d2a888dbc9a34bc695bb08bf21218c12) Notion page.
- User story: https://github.com/hirosystems/docs/issues/126.

### CODE REVIEW

File: docs/introduction.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+## What is Clarinet?
+
+[Clarinet](https://www.hiro.so/clarinet) provides a CLI package with a [clarity](https://clarity-lang.org/) runtime, an REPL, and a testing harness. Clarinet includes a Javascript library, testing environment, and a browser-based Sandbox. With Clarinet, you can rigorously iterate on your smart contracts locally before moving into production.
+
+Clarinet consists of two components:
+
+- Clarity REPL (Read, Evaluate, Print, Loop)
+- Testing harness.
+
+When the above components are used together, you can rapidly develop and test a Clarity smart contract, with the need to deploy the contract to a local devnet or testnet environments.
```
Comment (obycode) [inline docs/introduction.md:15]: I think these two bullet points are too limited. Should we just let the above paragraph cover everything?

File: docs/introduction.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+[![Clarinet101](images/clarinet101.png)](https://youtube.com/playlist?list=PL5Ujm489LoJaAz9kUJm8lYUWdGJ2AnQTb) 
+
+For more latest information on Clarinet product, refer to [blog posts on Clarinet](https://www.hiro.so/search?query=Clarinet).
+
+The Clarinet tool is used for developing smart contracts using a larger development strategy that involves:
+
+- Building and testing the contract locally.
+- Deploying the final draft contract to a testnet environment.
+- Testing on a live blockchain.
+- Deploying the final contract to the mainnet.
```
Comment (obycode) [inline docs/introduction.md:31]: This just feels like a repetition of what was noted above already.

**Review by obycode — COMMENTED** [review]
Thanks for these updates. I've added some comments to refine.

File: docs/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+```
+git pull
+git submodule update --recursive
+```
+
+
+Now that you have installed and built Clarinet, you can [create a new project](how-to-guides/how-to-create-new-project.md), [add a new contract](how-to-guides/how-to-add-contract.md) and then populate the project with smart contracts.
+
+Clarinet also provides tools for interacting with your contracts in a Read, Evaluate, Print, Loop (REPL) console, and perform automated [testing of contracts](how-to-guides/how-to-test-contract.md).
```
[the remark this answers is not in the document]
Comment (LakshmiLavanyaKasturi) [reply docs/getting-started.md:114]: Agree that we need a `How-to` page if we have elaborated steps to use the console. Added issue: https://github.com/hirosystems/docs/projects/1#card-87157201

File: docs/introduction.md (hunk 2 above)
Comment (LakshmiLavanyaKasturi) [reply docs/introduction.md:31]: Reworded it.

File: docs/getting-started.md [hunk 4]
```diff
@@ -66,10 +66,10 @@ mv ./clarinet /usr/local/bin
 >**_NOTE:_**
 >
 >If you are using macOS, you may receive security errors when trying to run the pre-compiled binary. 
->You can resolve the security warning by using the command below.
+>In the command mentioned below, update the <path-to-the-downloaded-client-library> with your library path and then use the command to resolve the security warning.
 
 ```sh
-xattr -d com.apple.quarantine /path/to/downloaded/clarinet/binary
+xattr -d com.apple.quarantine <path-to-the-downloaded-client-library>
```
Comment (obycode) [inline docs/getting-started.md:72]: "library path" is still confusing. How about:

To resolve this error, use the command below, replacing `<path-to-clarinet-binary>` with the path to the downloaded `clarinet` binary (e.g. `/usr/local/bin/clarinet`).

```sh
xattr -d com.apple.quarantine <path-to-clarinet-binary>
```

**Review by obycode — COMMENTED** [review]
One more comment.

File: docs/getting-started.md (hunk 4 above)
Comment (LakshmiLavanyaKasturi) [reply docs/getting-started.md]: Done

**Comment by LakshmiLavanyaKasturi** [comment]
@obycode Thanks for your comments. I made changes based on your comments. Please let me know if this PR is good to go. 

Also, just to note, If there are any further documentation related changes, I will take them in my next PR's. Since we are just addressing remote content issue in this PR. Thanks!

**Review by obycode — APPROVED** [review]
Ok, thanks. As mentioned, let's go ahead and merge this for the structure and address any documentation specific changes later. Thanks!

**Pull request merged**